keyboards/kopibeng/mnk65/mnk65.h

```diff
@@ -45,11 +51,11 @@
     K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K2C, K1C,      K1E, \
```

Comment:
Since you added matrix positions to info.json, the C layout macro should be removed.